docs/setup/aws.md

```diff
@@ -224,8 +224,6 @@ Depending on which file system you use, please add the following dependencies. Y
 - `S3AFileSystem`:
   - `hadoop-aws-2.7.2.jar`
   - `aws-java-sdk-1.7.4.jar`
-  - `httpcore-4.2.5.jar`
-  - `httpclient-4.2.5.jar`
```

Comment:
I see your point. Yeah, users run on AWS EMR don't necessarily use Kinesis. I'll revert this part.